roles/etcd/tasks/main.yml

```diff
@@ -37,6 +37,12 @@
     mode: 0700
   when: etcd_is_thirdparty | bool
 
+- name: Create configuration file in sysconfig
+  file:
+    path: /etc/sysconfig/etcd
+    state: touch
+  when: etcd_is_thirdparty | bool
```

Comment:
It's not necessary, but I think it would make sense since it's the same condition for the creation of the unit file that needs it